src/tests/ftest/soak/soak.py

```diff
@@ -131,21 +127,23 @@ def setUp(self):
             if host_server in self.hostlist_clients:
                 self.hostlist_clients.remove(host_server)
                 self.exclude_slurm_nodes.append(host_server)
+
+        # Include test node for log cleanup; remove from client list
+        local_host_list = include_local_host(None)
+        self.exclude_slurm_nodes.extend(local_host_list)
+        if local_host_list[0] in self.hostlist_clients:
+            self.hostlist_clients.remove((local_host_list[0]))
         self.log.info(
             "<<Updated hostlist_clients %s >>", self.hostlist_clients)
         if not self.hostlist_clients:
             self.fail("There are no nodes that are client only;"
                       "check if the partition also contains server nodes")
-
-        # Include test node for log cleanup; remove from client list
-        local_host_list = include_local_host(None)
-        self.exclude_slurm_nodes.extend(local_host_list)
-
-        # Start an agent on the test control host to enable API calls for
-        # reserved pool and containers.  The test control host should be the
-        # last host in the hostlist_clients list.
-        agent_groups = {self.server_group: local_host_list}
-        self.start_agents(agent_groups)
+        # create dmg command object for soak
+        dmg_config_file = self.get_config_file("daos", "dmg")
+        dmg_cfg = DmgYamlParameters(
+            dmg_config_file, self.server_group, DmgTransportCredentials())
+        dmg_cfg.hostlist.update([self.hostlist_servers[0]])
+        self.dmg = DmgCommand(self.bin, dmg_cfg)
```

Comment:
Maybe we should include this in `self.get_dmg_command()` if `self.server_managers` isn't defined.